content/blog/2024-01-01-flash-intro.Rmd

```diff
@@ -0,0 +1,37 @@
+---
+title: Introducing the FlaSH Group
+author: Ananya Joshi, Nolan Gormley, Richa Gadgil, Tina Townes
+date: 2024-01-01
+tags:
+  - flash
+authors:
+  - ananya
+  - nolan
+  - richa
+  - tina
+heroImage: flash_long.png
+heroImageThumb: flash_logo.png
+summary: | 
+  Delphi's FlaSH group works to alert experts about data that suggests quality issues or changes in disease dynamics from the millions of data points they publish daily to improve the efficacy of Delphi's data.
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+Delphi publishes millions of public-health-related data points per day, including the total number of daily influenza cases, hospitalizations, and deaths per county and state in the United States (US). This data helps public health practitioners, data professionals, and members of the public take important, informed decisions relating to health and well-being. 
+
+Yet, as data volumes continue to grow quickly (Delphi's data volume expanded 1000x in just 3 years), it is infeasible for data reviewers to inspect every one of these data points for subtle changes in  
+* quality (like those resulting from data delays) or  
+* disease dynamics (like an outbreak).  
```

Comment:
these bullets arent rendering properly...  maybe try a space in front of them?  if that doesnt work, they might need newlines around them?
```suggestion

 * quality (like those resulting from data delays) or
 * disease dynamics (like an outbreak).

```